### ISSUE TITLE
GAPIT.Association.GWAS.results

### ISSUE THREAD

**Issue by 9437951970** [jiabowang/GAPIT#100]
I am getting same result using "MLM","CMLM", "MLMM","Enriched CMLM" model. In GAPIT there are total 10 models. Is there any scenario, when the above mentioned models give same result in GWAS.

**Comment by jiabowang** [comment]
No. Could you please send your data to me? So I can help to figure the reason.

**Comment by 9437951970** [comment]
Dear sir, please find the attached data files. I have got the same result
for *MLM* and *ECMLM *in GWAS association results file .

Regards
Ashis
*------------------------------------------------------------------------------------------------------*
*Ashis Ranjan Udgata*
Scientist, Social Science division,
ICAR-Indian Institute of Pulse Research,
Kalyanpur, Kanpur (UP), India, 208024



“SAVE TREE - Please do not print this e-mail unless necessary.”

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
03/01/24,
09:40:49 AM

**Comment by jiabowang** [comment]
That may be reasonable. The compress does not work for some conditions, and the number of individuals was optimum for compress groups.

**Comment by 9437951970** [comment]
Thank you for your quick response.

Regards
Ashis
*------------------------------------------------------------------------------------------------------*
*Ashis Ranjan Udgata*
Scientist, Social Science division,
ICAR-Indian Institute of Pulse Research,
Kalyanpur, Kanpur (UP), India, 208024



“SAVE TREE - Please do not print this e-mail unless necessary.”

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
03/01/24,
11:26:40 AM
